aws/ecs_cluster.go

```diff
@@ -11,14 +11,54 @@ import (
 	"github.com/gruntwork-io/gruntwork-cli/errors"
 )
 
+// Used in this context to determine if the ECS Cluster is ready to be used & tagged
+// For more details on other valid status values: https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#Cluster
+const activeEcsClusterStatus string = "ACTIVE"
+
+// Filter all active ecs clusters
+func getAllActiveEcsClusterArns(awsSession *session.Session) ([]*string, error) {
+	svc := ecs.New(awsSession)
+
+	allClusters, err := getAllEcsClusters(awsSession)
+	if err != nil {
+		logging.Logger.Errorf("Error getting all ECS clusters")
+		return nil, errors.WithStackTrace(err)
+	}
+
+	var filteredEcsClusterArns []*string
+
+	batches := split(aws.StringValueSlice(allClusters), 10)
```

Comment:
NIT: Why batches of 10? Might want to extract that "magic number" into a `const` with a comment explaining the reasoning.

Reply:
😆 good question - I followed strictly the example Yori referred me to. What you've suggested makes sense.